personalitytest/static/js/src/personalitytest-view.js

```diff
@@ -238,6 +238,7 @@ function PersonalityTestXBlockStudent(runtime, element) {
         var handlerUrl = runtime.handlerUrl(element, 'reset_answers');
         $.post(handlerUrl, '{}')
             .done(function () {
+                $('select', element).find('option:selected').removeAttr('selected');
```

Comment:
Ow I thinked it was the first post (the one with the problem), not a answer.
My mistake ^^